### PR TITLE
test: attempt fix a flaky test

### PR DESCRIPTION
**Pull request by omBratteng**
This test keeps being flaky locally, so this should fix it hopefully.
Only did this specific one as it was the most flaky and annoyed me

### CODE REVIEW

File: __tests__/feeds.ts [hunk 1]
```diff
@@ -2276,7 +2276,21 @@ describe('function feedToFilters', () => {
...
     loggedUser = '1';
     await saveFeedFixtures();
-    expect(await feedToFilters(con, '1', '1')).toMatchSnapshot();
+    const filters = await feedToFilters(con, '1', '1');
+    expect(filters.blockedContentCuration).toEqual([]);
+    expect(filters.excludeTypes).toEqual([]);
+    expect(filters.sourceIds).toEqual([]);
+    expect(filters.includeTags).toEqual(
+      expect.arrayContaining(['html', 'javascript']),
+    );
```
Comment (capJavert) [inline __tests__/feeds.ts:2285]: In addition to `arrayContaining`  maybe also check length so we are sure no additional elements are added and test does not catch it?

Comment (capJavert) [reply __tests__/feeds.ts:2285]: also below

**Review by capJavert — APPROVED** [review]
Approving aside from above comment 👍

File: __tests__/feeds.ts (hunk 1 above)
Comment (omBratteng) [reply __tests__/feeds.ts:2285]: 👍

**Pull request merged**